pillar/components/execution.sls

```diff
@@ -23,7 +23,7 @@ execution:
       - system.storage
       - misc_pkgs.rsyslog
       - system.logrotate
-    3rd_party:
+    prerequisites:
```

Comment:
Kindly check where do we incorporate foundation?

Reply:
please check first comment